assets/styles/stylesl-16.css

```diff
@@ -0,0 +1,144 @@
+html, body {
+    width: 100vw; 
+    height: 100vh; 
+    background-color: rgb(30, 103, 170);
+}
+.grid-container {
+    display: grid;
+    width: 100vw;
+    height: 100vh;
+    grid-template-columns: 100px 1fr 150px;
+    grid-template-rows: 90px repeat(3, 1fr) 90px;
+    gap: 5px;
+    margin: 5px ;
```

Comment:
no margin needed. 
but you can add 10px padding to get the blue border around the page